etcdserver/apply.go

```diff
@@ -132,6 +132,9 @@ func (a *applierV3backend) Apply(r *pb.InternalRaftRequest) *applyResult {
 	ar := &applyResult{}
 	defer func(start time.Time) {
 		warnOfExpensiveRequest(a.s.getLogger(), start, &pb.InternalRaftStringer{Request: r}, ar.resp, ar.err)
+		if ar.err != nil {
```

Comment:
Wondering how much does this affect server throughput. Maybe use debug level to print?

Reply:
The logging overhead will be introduced only for failed requests, so I feel adding the warning log would help troubleshooting.

Reply:
yes ,i agree with @mitake. if there are too much failed log,something may be wrong with the cluster and users can also adjust the log leve to level. @jingyih